client-templates/github/accept.json.sample

```diff
@@ -114,6 +126,13 @@
       "origin": "https://${GITHUB_TOKEN}@${GITHUB_API}"
     },
 
+    {
+      "//": "used to create a new branch messages",
```

Comment:
~messages~

Reply:
Do we not do this today via `github-pr`? I wonder how brokered PRs are being opened.

Reply:
we replaced the github api library from https://github.com/pksunkara/octonode to https://github.com/octokit/rest.js